packages/knip/test/plugins/msw.test.ts

```diff
@@ -14,15 +14,15 @@ test('Should not see the msw files in issues', async () => {
     isStrict: true,
   });
 
-  assert.equal(issues.files.size, 3);
+  assert.equal(issues.files.size, 13);
   assert(issues.files.has(join(cwd, 'junk.js')));
   assert(issues.files.has(join(cwd, 'mocks/junk.js')));
   assert(issues.files.has(join(cwd, 'src/mocks/junk.js')));
 
   assert.deepEqual(counters, {
     ...baseCounters,
     devDependencies: 1,
-    files: 3,
+    files: 13,
```

Comment:
Please remove unused files, we don't need to assert that functionality in this test.